From 7e502573c104199797458bfbf5b4d665d3ec4b78 Mon Sep 17 00:00:00 2001
From: Christine Gharzuzi <chrisg@marvell.com>
Date: Wed, 10 May 2017 12:26:28 +0300
Subject: [PATCH] arm64: mvebu: a8k: add clock info prints

This patch override print_cpuinfo to print clock info
during boot.

Change-Id: I5d42c10123c6983c452b248d969f505a42153d06
Signed-off-by: Christine Gharzuzi <chrisg@marvell.com>

---
 arch/arm/mach-mvebu/armada8k/cpu.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm/mach-mvebu/armada8k/cpu.c b/arch/arm/mach-mvebu/armada8k/cpu.c
index d0760cafab..e29c1d39f1 100644
--- a/arch/arm/mach-mvebu/armada8k/cpu.c
+++ b/arch/arm/mach-mvebu/armada8k/cpu.c
@@ -12,6 +12,7 @@
 #include <asm/arch/cpu.h>
 #include <asm/arch/soc.h>
 #include <asm/armv8/mmu.h>
+#include <mach/clock.h>
 #include <mach/soc.h>
 
 /* Armada 7k/8k */
@@ -155,6 +156,7 @@ int mmc_get_env_dev(void)
 int print_cpuinfo(void)
 {
 	soc_print_device_info();
+	soc_print_clock_info();
 	return 0;
 }
 #endif
